diff --git a/src/common/logging.cpp b/src/common/logging.cpp
index 3022c8d8d..ba70b38da 100644
--- a/src/common/logging.cpp
+++ b/src/common/logging.cpp
@@ -106,7 +106,7 @@ void otbrLogInit(const char *aProgramName, otbrLogLevel aLevel, bool aPrintStder
         ident = strrchr(aProgramName, '/');
         ident = (ident != nullptr) ? ident + 1 : aProgramName;
 
-        openlog(ident, (LOG_CONS | LOG_PID) | (aPrintStderr ? LOG_PERROR : 0), OTBR_SYSLOG_FACILITY_ID);
+        openlog(ident, LOG_PID | (aPrintStderr ? LOG_PERROR : 0), LOG_USER);
     }
 #endif
 
